app/models/student.rb

```diff
@@ -43,7 +43,7 @@ def current_team
 
   def current_drafts
     @current_drafts ||= if current_team
-      current_team.application_drafts.in_current_season
+                          current_team.application_drafts.in_current_season
```

Comment:
I think it is, but I cannot name it. I really like this way of indentation. Vim's Ruby syntax auto-formats blocks like this in exactly the same way.

Reply:
It's [this cop](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/ElseAlignment) and I'm gonna add it, since it otherwise conflicts with strict 2-spaces indentation anyways (like here...)